repl.html

```diff
@@ -1,166 +1,167 @@
 <!DOCTYPE html>
 <html>
-<head>
-<meta charset="utf-8">
-<meta name="viewport" content="width=device-width, initial-scale=1">
-<title>Scrapscript Web REPL</title>
-<link rel="shortcut icon" type="image/x-icon" href="data:image/vnd.microsoft.icon;base64,AAABAAEAEBAQAAAAAAAoAQAAFgAAACgAAAAQAAAAIAAAAAEABAAAAAAAgAAAAAAAAAAAAAAAEAAAAAAAAAD/AAAA////AAAA/wAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQAAAAAAAREBEQAAAAAQAQEAEAAAABABAQAQAAAAAREBEQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACIiIiIiIiIiIiIiIiIiIiIiERIiESEiIiIiESEiISIiIhEiISIhEiIiIREiESEhIiIiIiIiIiIiIiIiIiIiIiIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA">
-<link rel="preconnect" href="https://fonts.googleapis.com" />
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-<link
-  href="https://fonts.googleapis.com/css2?family=Fira+Code&display=swap"
-  rel="stylesheet"
-/>
-<link
-  href="https://fonts.googleapis.com/css2?family=Rubik:ital,wght@0,500;0,700&display=swap"
-  rel="stylesheet"
-/>
-<link
-  href="https://fonts.googleapis.com/css2?family=Nunito+Sans:ital,wght@0,400;0,700;0,900;1,400&display=swap"
-  rel="stylesheet"
-/>
-<link rel="stylesheet" href="/style.css" />
-</head>
-<body>
-<main>
-<div>
-<p>See <a href="https://scrapscript.org/">scrapscript.org</a> for a slightly
-out of date language reference.</p>
-<p>This REPL sends programs to a server to be evaluated, but the server is
-completely stateless. All persistence is in the browser.</p>
-</div>
-<div>
-<!-- TODO(max): Add button to save to/load from disk. -->
-<button id="clear-local-storage">Clear LocalStorage</button>
-</div>
-<div id="output" style="height: 400px; overflow: auto;">
-Output:
-</div>
-<div>
-<code>>>> </code><input id="input" type="text" />
-</div>
-<script type="module">
-"use strict";
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Scrapscript Web REPL</title>
+    <link
+      rel="shortcut icon"
```

Comment:
there's a lot of other stuff happening in the commit which makes it hard to see what is actually going on